app/selectors.js

```diff
@@ -239,8 +239,17 @@ export const hasUnminedTransactions = compose(l => l && l.length > 0, get(["grpc
 export const transactions = createSelector(
   [transactionsNormalizer, get(["grpc", "transactions"])], apply
 );
+
 export const homeHistoryTransactions = createSelector(
-  [transactionsNormalizer, get(["grpc", "recentTransactions"])], apply
+  [transactions],
```

Comment:
If you use `transactions` instead of `get(["grpc", "recentTransactions"])`, changes to the current transaction filter will change the transactions shown in the overview page. Is that what you want to accomplish or are you going to (after this PR) change the behaviour of the transactions selector? If not, then you need to return it (the `grpc.recentTransactions` state is only filled during startup and after a successful rescan).